packages/runtime/src/util/renderMuiIcons.tsx

```diff
@@ -0,0 +1,20 @@
+import * as MuiIcons from "@mui/icons-material";
+
+export const renderMuiIcon = (
```

Comment:
Make this an Icon component instead of a util function

Reply:
@evshi do you mean Icon widget?

Reply:
Hmm, actually there's already a util for this see `Icon.tsx`

Reply:
@sanaf1333 can you address this when you can a chance?

Reply:
@evshi  fixed this, now using current Icon component for this instead of a function.